packages/web/components/table/types.ts

```diff
@@ -1,42 +1,7 @@
-import { ReactElement } from "react";
-
-import { CustomClasses, SortDirection } from "~/components/types";
-import { Breakpoint } from "~/hooks";
-
 export interface BaseCell {
```

Comment:
It does not but we are using them in AssetCell that is a component we are using in a react table. We can remove them without losing functionality but it would mean to change some other components as well. Do you want me to it in this PR or in a new one?

Reply:
At the end it was really used on a single component and was there as a dead extension for all the other ones so I changed the said component and removed it entirely